types/jsdom/base.d.ts

```diff
@@ -2,7 +2,7 @@
 /// <reference types="node" />
 
 import { EventEmitter } from "events";
-import { ElementLocation } from "parse5";
+import { Location } from "parse5/dist/common/token";
```

Comment:
note that `parse5` uses `import {type Thing}` syntax, so a minimum of TS 4.5 is required

Reply:
Add a comment like this one: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d561758c4ccf256e156cbff44943f665bb07e2cf/types/jest/index.d.ts#L31

Reply:
done in 1e6cfc30bd4b4796d3bc519a4fbc98558acd904d